lib/ES/Template.pm

```diff
@@ -50,12 +50,6 @@ sub apply {
 
         my $contents = $source->slurp( iomode => '<:encoding(UTF-8)' );
 
-        # Strip XML guff
```

Comment:
I'm slowly, slowly removing everything from this file. Internationally. Making the templating process as simple as possible will allow us much more flexibility about *when* we do it.